fjord/heartbeat/tests/test_api.py

```diff
@@ -154,6 +154,30 @@ def test_complete_answer(self):
 
         eq_(ans.is_test, False)
 
+    def test_country_unknown(self):
+        """This is a stopgap fix for handling 'unknown' value for country"""
+        survey = SurveyFactory.create()
+
+        data = {
+            'experiment_version': '1',
+            'response_version': 1,
+            'person_id': 'joemamma',
+            'survey_id': survey.name,
+            'flow_id': '20141113',
+            'question_id': '1',
+            'updated_ts': self.timestamp(),
+
+            'question_text': 'ou812?',
+            'variation_id': '1',
+            'country': 'unknown'
+        }
+        resp = self.client.post(
+            reverse('heartbeat-api'),
+            content_type='application/json',
+            data=json.dumps(data))
+
+        eq_(resp.status_code, 201)
```

Comment:
I'm sure you know this, but I want to write it down. This is testing only that no errors are thrown, not that the data gets translated to `UNK` or that the data gets kept at all. Which seems fine to me considering the scope of the problem and fix.